Will not install

I cannot get the app to install on my system. I run the command and it returns the following error:

C:\WINDOWS\system32>dotnet tool install --global haveibeenpwned-downloader
C:\Users\someuser\AppData\Local\Temp\jpq2nno2.b2t\restore.csproj : error NU1100: Unable to resolve 'haveibeenpwned-down
loader (>= 0.0.0)' for 'net6.0'.
C:\Users\someuser\AppData\Local\Temp\jpq2nno2.b2t\restore.csproj : error NU1100: Unable to resolve 'haveibeenpwned-down
loader (>= 0.0.0)' for 'net6.0/any'.
The tool package could not be restored.
Tool 'haveibeenpwned-downloader' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool

Any idea what the issue with the install command is?

--- a/README.md
+++ b/README.md
@@ -8,7 +8,8 @@ You'll need to install [.NET 6](https://dotnet.microsoft.com/en-us/download/dotn
 
 ## How to install
 1. Open a command line window
-2. Run `dotnet tool install --global haveibeenpwned-downloader`
+2. Run `dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org`
+3. Run `dotnet tool install --global haveibeenpwned-downloader`
 
 # Usage
 

--- a/README.md
+++ b/README.md
@@ -8,8 +8,13 @@ You'll need to install [.NET 6](https://dotnet.microsoft.com/en-us/download/dotn
 
 ## How to install
 1. Open a command line window
-2. Run `dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org`
-3. Run `dotnet tool install --global haveibeenpwned-downloader`
+2. Run `dotnet tool install --global haveibeenpwned-downloader`
+
+### Troubleshooting
+If the installer is unable to resolve the package, then you can run the following and then try again.
+```
+dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+```
 
 # Usage
 

